pkg/ccl/changefeedccl/encoder_json.go

```diff
@@ -31,6 +32,18 @@ import (
 	"github.com/cockroachdb/errors"
 )
 
+const (
+	// jsonNullAsObjectKey is the key used in null-as-object mode to represent a JSON null value.
+	jsonNullAsObjectKey = `__crdb_null_json__`
+)
+
+var jsonNullAsObjectJSONNullObj = sync.OnceValue(func() json.JSON {
+	j, _ := json.MakeJSON(map[string]any{
```

Comment:
@reviewer: this error should not be possible but i'm open to discussing alternatives